src/main/java/org/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatter.java

```diff
@@ -23,6 +23,9 @@ public String format(String text) {
             return result;
         }
 
+        NormalizeUnicodeFormatter normalizer = new NormalizeUnicodeFormatter();
```

Comment:
Make this a class constant: `public static final ...`.

Open the context menu, select "Refactor..." then "Introduce Field..."

![image](https://github.com/JabRef/jabref/assets/1366654/d6cf4f99-5bbd-41ae-b0f4-48bb0a2c7652)

Choose "field declaration" for initialization:

![image](https://github.com/JabRef/jabref/assets/1366654/09afbcc1-8eb5-4ef4-83f3-439d36c78aaa)

You will then have to make the variable name upper case. You can use <kbd>Alt</kbd>+<kbd>Enter</kbd> to get IntelliJ support:

![image](https://github.com/JabRef/jabref/assets/1366654/897d1571-f13c-4f38-a326-e0169d2d3405)